### PR TITLE
data : sandramsc

### PR DESCRIPTION
**Pull request by sandramsc**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5576"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/sandramsc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Sandra Ashipala",
+  "displayStatsPublic": "true",
```
[the remark this answers is not in the document]
Comment (sandramsc) [reply data/sandramsc.json:3]: this was in the YouTube video - Quickstart

File: data/sandramsc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "Software developer, contributing to the frontend/backend for building web apps.",
+  "avatar": "https://github.com/sandramsc.png",
```
[the remark this answers is not in the document]
Comment (sandramsc) [reply data/sandramsc.json:6]: this was in the YouTube video - Quickstart

how can the documentation be updated so each newcomer knows not to add this _"avatar": "https://github.com/username.png",_ line?

[the remark this answers is not in the document]
Comment (priyankarpal) [reply data/sandramsc.json:6]: > this was in the YouTube video - Quickstart
> 
> how can the documentation be updated so each newcomer knows not to add this _"avatar": "https://github.com/username.png",_ line?

after recent update these are not needed.

**Review by priyankarpal — APPROVED** [review]
Awesome Looks Great 🎉

**Review by accodes21 — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]
Look Good! Thanks for your contribution.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**